proposals/lip-0053.md

```diff
@@ -54,23 +54,22 @@ The `inboxUpdate` contains the information relative to the messages to be includ
 
```

Comment:
> Used to update the cross-chain account. It must be a certificate with an adequate [BLS signature](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md). The signature is always validated against the public key set stored in the sending chain account.

The last sentence is not correct as there is no sending chain account and there are multiple public keys and weights.

Reply:
>  If newCertificateThreshold is set to 0, it implies that the certificate threshold has not been changed in the sending chain.

This seems like an odd rule that requires more complex logic with easier mistakes. Why not always put the actual certificate threshold here, independently if is changes or not.

Reply:
> As specified in [LIP 0045](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md), cross-chain messages are stored in the chain outbox on the sending chain and in the corresponding inbox on the receiving chain. This property contains three elements: the cross-chain messages themselves, an inclusion witness into the outbox root, and an inclusion witness of this outbox root into the state root.

- "cross-chain messages are stored in the chain outbox": Do we actually define a chain outbox (e.g., similar to the transactions or block store) and a chain inbox? Those would be not part of the state, but the state transition history.
- "inclusion witness into the outbox root": I think it should be "with respect to the outbox root"
- "an inclusion witness of this outbox root into the state root": I think it should be "with respect to the state root"

Reply:
I think there are multiple occurrences of "an inclusion proof into" that should be changed to "an inclusion proof with respect to".

Reply:
- Fixed the validators sentence.
- sounds good, I will update the specs
- changed the sentence to clarify a bit
- Fixed

Reply:
Looks good.